packages/Checkbox/src/Checkbox.tsx

```diff
@@ -64,22 +71,43 @@ const Checkbox: React.FunctionComponent<CheckboxProps> = React.forwardRef(
       setStatus(!status);
     };
 
+    const showLabel = appearance === 'completion' || children;
+
     return (
-      <span className={cx('ids-form-control', className)}>
+      <span
+        className={cx(
+          'ids-form-control',
+          className,
+          `ids-form-control--${appearance}`
+        )}
+      >
         <input
           ref={checkRef}
           id={htmlFor}
-          className="ids-checkbox"
+          className={cx('ids-checkbox', `ids-checkbox--${appearance}`)}
```

Comment:
I would slightly change the code here to avoid having `ids-checkbox--default`

```suggestion
          className={cx('ids-checkbox', {
            [`ids-checkbox--${appearance}`]: appearance !== 'default',
          )}
```